README.md

```diff
@@ -18,16 +18,22 @@ The following instruction assume your are using Python 2 and an Ubuntu installat
 
 ## Dependencies
 
-Pymavlink requires two dependencies :
+Pymavlink requires 3 dependencies :
 
-    - future : for python 2 and python 3 interoperability
-    - lxml : for checking and parsing xml file
+    - future : for python 2 and python 3 interoperability (http://python-future.org/)
+    - lxml : for checking and parsing xml file (http://lxml.de/installation.html)
 
 ### On linux
 
-Using pip you can install the required dependencies
+Lxml require some dependencies too
 
 ```bash
+sudo apt-get install libxml2-dev libxslt-dev python-dev
```

Comment:
Should mention this is for debian-derived systems and the user should adjust for their particular package manager.